CHANGELOG.md

```diff
@@ -2,7 +2,30 @@
 
 ## Unreleased
 
-Unreleased: https://github.com/salemove/ios-sdk/compare/0.29.4...master
+Unreleased: https://github.com/salemove/ios-sdk/compare/0.30.0...master
+
+## [0.30.0] - 2021-07-26
+
+### Added
+
+- Added push notification support
+- Added possibility start engagement with media direction
```

Comment:
I think this one is not needed anymore, as the `Added possibility to start engagement with media` covers it.

Reply:
fixed cc9f93bf3c5d4fa196644a09f9d1b9e3c7430184